README.md

```diff
@@ -1 +1,171 @@
-# 1-capstone-project-module-1
\ No newline at end of file
+# 1-capstone-project-module-1
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 CC GLOBAL SUMMIT 2023 WEBSITE <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**CC GLOBAL SUMMIT PROJECT** In this capstone project based on an online website for a conference. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript & DOM</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Added semantic HTML tags.**
+- **Added CSS box model**
+- **Added JavaScript to manipulate DOM elements and events.**
+- **Added objects to store and access data.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [live project](https://shri-sakthi-v.github.io/1-capstone-project-module-1/index.html)
+
+## 🚀 Live Video <a name="live-video"></a>
+
+- [live video ](https://www.loom.com/share/786d31bc4d07425081894799f3d5bcd8)
+
```

Comment:
**[REQUIRED CHANGES]**

:exclamation: :exclamation: **VIDEO MAXIMUM LENGTH EXCEEDED** :exclamation: :exclamation: 

- Great job with the video, according to the previous reviewer's suggested change :point_down: 

>  In your video presentation, you explained what is your website about and its features but you forgot to mention a piece of code that you are proud of which is very important as it shows your ability to explain technical concepts clearly wink. Please record another video and include that part too :smiley:  :+1: :point_up_2: 


however, you've exceeded the video's maximum length which should not be more than 5 min. You'd have to record another video meeting all the video requirements. please, [Check the project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md).

Reply:
thank you for notifying me, of the change made.